src/site/twitch.tv/modules/stream-info/StreamInfoModule.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<Teleport v-if="displayLatency" to="#seventv-stream-info">
+		<figure><GaugeIcon /></figure>
+		<span v-if="shouldShowVideoStats" class="seventv-stream-info">{{ videoLatency }}s</span>
+	</Teleport>
+</template>
+
+<script setup lang="ts">
+import { declareConfig, useConfig } from "@/composable/useSettings";
+import { declareModule } from "@/composable/useModule";
+import { onMounted, onUnmounted, ref } from "vue";
+import GaugeIcon from "@/assets/svg/icons/GaugeIcon.vue";
+import { useComponentHook } from "@/common/ReactHooks";
+
+const { markAsReady } = declareModule("stream-info", {
+	name: "Stream Info",
+	depends_on: [],
+});
+
+const shouldShowVideoStats = useConfig<boolean>("channel.latency_info");
+const displayLatency = ref<boolean>(false);
+const videoLatency = ref<string>("");
+
+const mediaPlayerComponent = useComponentHook<Twitch.MediaPlayerInstanceComponent>({
+	predicate: (el) => el.props && el.props.mediaPlayerInstance,
+});
+
+// This does not render instaneously
+// Need to find alternative hook for rendering the stream-info element
+useComponentHook<ReactExtended.AnyReactComponent>(
+	{
+		predicate: (el) => el.props && el.instance,
+	},
+	{
+		hooks: {
+			render(inst, cur) {
+				if (
+					inst.component?.instance?.d?.parentElement?.attributes?.getNamedItem("aria-label")?.nodeValue !==
+					"Viewer Count"
```

Comment:
This will not work on non-english languages

Reply:
Not Applicable anymore